python/ray/includes/metric.pxi

```diff
@@ -0,0 +1,100 @@
+from ray.includes.metric cimport (
+    CCount,
+    CGauge,
+    CHistogram,
+    CTagKey,
+    CSum,
+    CMetric,
+)
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string as c_string
+from libcpp.unordered_map cimport unordered_map
+from libcpp.vector cimport vector as c_vector
+
+cdef class TagKey:
+    cdef c_string __name
+    def __init__(self, name):
+      self.__name = name.encode("ascii")
+      CTagKey.Register(self.__name)
+    
+    def name(self):
+      return self.__name
```

Comment:
btw, the indentation seems to be wrong for some reasons. Do you know why?

Reply:
> btw, the indentation seems to be wrong for some reasons. Do you know why?

Perphas yapf doesn't work for cython files. I'll check all added files manually.